bcda/cclf/parser.go

```diff
@@ -41,8 +41,13 @@ type CSVParser struct {
 func getACOConfigs() ([]service.ACOConfig, error) {
 	configs, err := service.LoadConfig()
 	if err != nil {
+		log.API.Errorf("error when fetching configs: %s", err)
 		return []service.ACOConfig{}, err
 	}
+	if configs == nil {
```

Comment:
Should this logic check also return an error and an empty service config?

Reply:
These commits are purely for testing/logging (since I cannot recreate locally). I will be reverting these -- left the PR in draft mode and hadn't request review yet since I would be adding things like the above if I needed to troubleshoot.